console/v1/types_console_log_links.go

```diff
@@ -0,0 +1,23 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ConsoleLogLink is an extension for customizing OpenShift web console pod log links.
+type ConsoleLogLink struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ConsoleLogLinkSpec `json:"spec"`
+}
+
+// ConsoleLogLinkSpec is the desired console pod log link configuration.
+type ConsoleLogLinkSpec struct {
+	LogLink `json:",inline"`
+	// An optional, regular expression used to restrict a log link to a matching
+	// namespace (e.g., ^openshift-).
```

Comment:
You should mention that the links will be displayed for all namespaces if not specified

Reply:
Doc string should start with the JSON field name